src/oss/confluent.sh

```diff
@@ -1308,6 +1310,60 @@ extract_json_config() {
     _retval="${parsed_json}"
 }
 
+produce_command() {
+    local topicname="${1}"
+    local args="$@"
+
+    if [[ "x${topicname}" == "x" ]]; then
+        echo "Missing required topic name argument in '${command_name} produce <topicname>'"
+        produce_usage
+    fi
+
```

Comment:
suggest using local variables and in lower case for BROKERLIST and AVRO

Reply:
Done